assembly-syntax/src/ast/constants.rs

```diff
@@ -15,6 +15,7 @@ use crate::{
 // ================================================================================================
 
 /// Represents a constant definition in Miden Assembly syntax, i.e. `const.FOO = 1 + 1`.
+#[derive(Clone)]
 pub struct Constant {
```

Comment:
Not related to this PR, but I would love to add the ability to export constants from modules as well. The fact that we cannot do this right now requires pretty ugly workarounds (e.g., we wrap all constants in functions that just put these constants onto the stack).

Reply:
I did actually add that in this PR, via `Module::constants`. We can add some convenience functions in another PR, e.g. `Module::get_constant` (to look up a constant by name).

Reply:
Ah nice! To use this in MASM (i.e., to import constants from other modules), would we need to change anything in the parser (or anywhere else)?